src/integrationTest/resources/db/testmigration/V1_64__elinks_update_jrd_lrd_region_mapping_table.sql

```diff
@@ -0,0 +1,3 @@
+-- update row script : dbjudicialdata.jrd_lrd_region_mapping
+
+UPDATE jrd_lrd_region_mapping  SET jrd_region  = 'Yorkshire and the Humber' WHERE jrd_region_id  = '32';
```

Comment:
Hi @prudhvi-maddineni  please update the tabl eas dbjudicialdata.jrd_lrd_region_mapping